packages/react/src/primitives/shared/styleUtils.ts

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+import autoprefixer from 'autoprefixer';
+import postcssJs from 'postcss-js';
+
+export const prefixer = postcssJs.sync([autoprefixer]);
+
+import {
+  ComponentPropsToStylePropsMap,
+  ComponentPropsToStylePropsMapKeys,
+  GridItemStyleProps,
+  ResponsiveObject,
+  ViewProps,
+} from '../types/index';
+
+import { getValueAtCurrentBreakpoint } from './responsive/utils';
+import { useBreakpoint } from './responsive/useBreakpoint';
+import { Breakpoint, Breakpoints } from '../types/responsive';
+
+import { useTheming } from '../../theming/';
+import { filterOutNullOrEmptyStringValues, isNullOrEmptyString } from './utils';
+
+/**
+ * Transforms style props to another target prop
+ * where the original is a simpler API than the target.
+ * This function will remove the original prop and
+ * replace target prop values with calculated
+ * E.g. rowSpan => row, columnSpan => column
+ * @param props T
+ * @returns
```

Comment:
are these from another doc?

Reply:
This was a mistake. I've removed it since that information is in the Types